setup.md

```diff
@@ -113,3 +105,17 @@ you can run it by opening a terminal and typing `bash`.
 [gnome-terminal]: https://help.gnome.org/users/gnome-terminal/stable/
 [kde-konsole]: https://konsole.kde.org/
 [xterm]: https://en.wikipedia.org/wiki/Xterm
+
+
+>## Download and extract the data-shell.zip-file:
```

Comment:
I think the formatting is bad here? This is both a quote and a second-level title.